### ISSUE TITLE
pr command not working in Ventura 13.0 + Alfred 5.0.3 [2087]

### ISSUE THREAD

**Issue by iltaek** [edgarjs/github-repos-alfred-workflow#52]
When I try to use `pr` command, it is switched to web search mode.
> FYI. this is the OS version and I am using Alfred 5.0.3 [2087]
> ```
> $ sw_vers
> ProductName:		macOS
> ProductVersion:		13.0
> BuildVersion:		22A380
> ```
## `pr `
<img width="719" alt="CleanShot 2022-10-28 at 05 42 05@2x" src="https://user-images.githubusercontent.com/5857198/198393796-dfd67884-1038-45fe-b79b-ec91a455a317.png">

```
[05:48:54.424] Logging Started...
[05:49:00.731] GitHub Repos[Script Filter] Queuing argument ''
[05:49:01.381] GitHub Repos[Script Filter] Script with argv '' finished
[05:49:01.388] ERROR: GitHub Repos[Script Filter] Code 1: exception object expected
[05:49:01.389] ERROR: GitHub Repos[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0. in JSON:
exception object expected
```

## `pr asdf`
![CleanShot 2022-10-28 at 06 03 36@2x](https://user-images.githubusercontent.com/5857198/198397814-16e764d4-80a0-42a0-b3ee-0d4ae42ce040.png)

```
[05:59:20.715] GitHub Repos[Script Filter] Queuing argument 'asdf'
[05:59:21.242] GitHub Repos[Script Filter] Script with argv 'asdf' finished
[05:59:21.251] ERROR: GitHub Repos[Script Filter] Code 1: exception object expected
```

**Comment by iltaek** [comment]
This is debug logs.

```
[05:48:54.424] Logging Started...
[05:49:00.731] GitHub Repos[Script Filter] Queuing argument ''
[05:49:01.381] GitHub Repos[Script Filter] Script with argv '' finished
[05:49:01.388] ERROR: GitHub Repos[Script Filter] Code 1: exception object expected
[05:49:01.389] ERROR: GitHub Repos[Script Filter] JSON error: JSON text did not start with array or object and option to allow fragments not set. around line 1, column 0. in JSON:
exception object expected
```

**Comment by dportalesr** [comment]
Seeing this too.

$ sw_vers
ProductName:		macOS
ProductVersion:		13.3
ProductVersionExtra:	(b)
BuildVersion:		22E7752300g

Alfred 5.0.6 [2110]

**Comment by roeniss** [comment]
Hi @edgarjs, I wonder whether this `alfred-github-repos` project has been abandoned or still maintained. 

If it's latter case, I want to take on this issue (and other I hope). 

So what's current state of this project? In other words, can you review my patch if i make pr?

**Comment by edgarjs** [comment]
Hi @roeniss, I've been a little busy with other projects at the moment. I'd appreciate if you can send a PR and I'll review it gladly. Thank you.

**Comment by roeniss** [comment]
I'm confused because I failed to reproduce this issue lol. I see no problem here. 

@dportalesr @iltaek Did you guys set your Personal Access Token properly with `gh-login $PAT` command? I think I didn't set it when I added the above comment. The token need to have `repo` section permissions.

**Comment by roeniss** [comment]
I'm not sure the exact reason but today succeded to reproduce with another computer, another organization. maybe kind of buffer overflow because some pr are shown and others are not.

**Comment by dportalesr** [comment]
@roeniss Yes, I have set it before. I also suspected it was something related to permissions so I created a new GH token but no luck. That newest token was configured like this
![Menubar-20230608-005917](https://github.com/edgarjs/alfred-github-repos/assets/237597/2d7009e9-029e-497b-8a46-b406cfc8a713)

**Comment by dportalesr** [comment]
I found this note in [Managing your personal access tokens](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens)
> Note: Organization owners can restrict the access of personal access token (classic) to their organization. If you try to use a personal access token (classic) to access resources in an organization that has disabled personal access token (classic) access, your request will fail with a 403 response. Instead, you must use a GitHub App, OAuth App, or fine-grained personal access token.

Now, I'm not so versed in Alfred workflows to verify if that's the issue.

**Comment by edgarjs** [comment]
A new version of the workflow has been published. It uses the GitHub CLI under the hood, so things should work better now. Please make sure to follow the instructions from the README.md to install the requirements and authenticate properly. I'm closing this issue for now but, let me know if things break.
